docs/python/guides/app.md

```diff
@@ -5,4 +5,14 @@ sidenav: side-nav-python.html
 type: markdown
 ---
 
-## `app.py`
\ No newline at end of file
+# `app.py`
+
+`app.py` is the generic entry point for Abseil Python applications.
```

Comment:
Think you should point out this is in lieu of using:

$python my_app.py" for example.